libycp/src/Import.cc

```diff
@@ -126,6 +135,13 @@ Import::import (const string &name, Y2Namespace *preloaded_namespace)
     }
     else
     {
+
+        if (isErrorNamespace(m_module->second))
+        {
+            //already failed to load, so just return 0
+            return 0;
```

Comment:
Can we also use `return -1/0` instead of the `exit` 10 lines below and remove a rpmlint warning?
```
 yast2-core.x86_64: W: shared-lib-calls-exit /usr/lib64/libycp.so.5.0.0 exit@GLIBC_2.2.5
```